graylog2-web-interface/src/components/common/Scratchpad.jsx

```diff
@@ -0,0 +1,181 @@
+import React, { useContext, useState, useEffect, useRef } from 'react';
+import styled, { css } from 'styled-components';
+import { rgba } from 'polished';
+import ClipboardJS from 'clipboard';
+
+import teinte from 'theme/teinte';
+import Interactable from 'components/common/Interactable';
```

Comment:
We mostly use named imports for importing common components, could you also put the `Icon` component imported down below in the same line, please?

Reply:
This is due to eslint cyclical import rule